modules/robin/src/test/ts/browser/zone/ZoneWalkerTest.ts

```diff
@@ -0,0 +1,49 @@
+import { after, before, describe, it } from '@ephox/bedrock-client';
+import { DomUniverse } from '@ephox/boss';
+import { Insert, Remove, SugarBody, SugarElement, SugarText } from '@ephox/sugar';
+import { assert } from 'chai';
+import { ZoneViewports } from 'ephox/robin/api/general/ZoneViewports';
+import * as ZoneWalker from 'ephox/robin/zone/ZoneWalker';
+
+const hook = (html: string) => {
+  let element: SugarElement<HTMLElement> | undefined;
+
+  before(() => {
+    element = SugarElement.fromHtml(html);
+    Insert.append(SugarBody.body(), element);
+  });
+
+  after(() => {
+    Remove.remove(element!);
+  });
+
+  return () => element!;
```

Comment:
Going to repeat what you asked me recently... instead of asserting this isn't nullable, maybe instead this should die if trying to access it when not ready?